site/en/workflows.md

```diff
@@ -778,6 +893,56 @@ This processor can be used to analyse the output of the OCR.
   </tbody>
```

Comment:
Wait, where's `ocrd-cor-asv-ann-evaluate`?

Reply:
It's in there now, even with a tip on how to capture STDERR.